.github/workflows/release.yml

```diff
@@ -62,7 +62,8 @@ jobs:
                 required_pull_request_reviews: null,
                 required_status_checks: {
                   checks: [
-                    { context: "build" },
+                    { context: "build (16.13.0)" },
+                    { context: "build (18.x)" },
```

Comment:
I think these is the right ids.
Having a space in there though makes me doubt though.

Reply:
Heh, yeah, I don't know. Guess we'll find out the hard way if we need! 😄